charts/pulsar/Chart.yaml

```diff
@@ -21,11 +21,11 @@ apiVersion: v1
 appVersion: "2.6.1"
 description: Apache Pulsar Helm chart for Kubernetes
 name: pulsar
-version: 2.6.1-2
+version: 2.6.1-3
 home: https://pulsar.apache.org
 sources:
-- https://github.com/apache/pulsar
+  - https://github.com/apache/pulsar
```

Comment:
Any ideas on these changes?

Reply:
sorry about that, It's from the prettier plugin in my vscode install, I was pretty sure I had it disabled for this workspace and was careless when adding the modifications. I have reverted the indentation changes